pkg/services/ngalert/api/api_alertmanager_base.go

```diff
@@ -18,15 +18,14 @@ import (
 )
 
 type AlertmanagerApiService interface {
-	RouteCreateSilence(*models.ReqContext, apimodels.CreateSilenceParams) response.Response
```

Comment:
This order changed in https://github.com/grafana/grafana/pull/32517
but I suggest keeping this order because this file is auto-generate so that we don't need to change on each API update